src/main/java/org/humancellatlas/ingest/file/File.java

```diff
@@ -87,12 +83,10 @@ public File addAsDerivedByProcess(Process process) {
 
```

Comment:
I will leave this here for now, but it looks like changing to Set makes processes contained in it unique anyway (depends on equality check impl)..